### PR TITLE
Reprocess merged copyrights

### PR DESCRIPTION
**Pull request by mnonnenmacher**
See the commit messages for details.

### CODE REVIEW

File: model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt [hunk 1]
```diff
@@ -123,14 +123,29 @@ data class ResolvedLicense(
...
     /**
      * Return all copyright statements associated to this license. Optionally [excludes][omitExcluded] copyright
-     * findings excluded by [PathExclude]s.
+     * findings excluded by [PathExclude]s. The licenses can optionally be [processed][process] using the
+     * [CopyrightStatementsProcessor].
      */
-    fun getCopyrights(omitExcluded: Boolean = false): Set<String> =
-        locations.flatMapTo(sortedSetOf()) { location ->
+    @JvmOverloads
+    fun getCopyrights(omitExcluded: Boolean = false, process: Boolean = false): Set<String> {
```
Comment (fviernau) [inline model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:130]: Would it make sense to swap the order of the parameters?
.. I'd expect that the likelyhood of `process = true` being passed much higher than `omitExcluded = true`.

Comment (fviernau) [reply model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:130]: Apart from that, how about making it `true` by default?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt (hunk 1 above)
Comment (mnonnenmacher) [reply model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:130]: It's the other way round, I expect that `omitExcluded` is almost always true. Reprocessing is only required if multiple licenses got merged into one which is something only the notice reporter does, I don't expect that if we add copyrights to any other reporter they would do that.
I will change the default value of `omitExcluded` and switch the order, this also simplifies the code in the templates.

Comment (fviernau) [reply model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:130]: yeah, that's what I meant actually. I just mixed up true with false for `omitExcluded`.

**Review by fviernau — APPROVED** [review]

**Pull request merged**